src/sentry/models/sentryappwebhookerror.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import EncryptedJsonField, FlexibleForeignKey, Model
+
+
+class SentryAppWebhookError(Model):
+    __core__ = False
+
+    date_added = models.DateTimeField(default=timezone.now)
+
+    sentry_app = FlexibleForeignKey("sentry.SentryApp", related_name="webhook_errors")
+
+    organization = FlexibleForeignKey(
+        "sentry.Organization", related_name="sentry_app_webhook_errors"
+    )
+
+    # If there is a related Sentry error, store it here
+    error_id = models.CharField(max_length=64, null=True)
+
+    request_body = EncryptedJsonField()
+
+    request_headers = EncryptedJsonField()
+
+    event_type = models.CharField(max_length=64)
+
+    # We need to store this rather than just fetch it from the related sentry app in case the URL is changed
+    webhook_url = models.URLField()
+
+    response_body = models.TextField()
```

Comment:
What's the max size of `models.TextField()`? I think we should limit the size of this to something reasonable. If someone sends a huge response back, we can just trim it.

Reply:
maybe we can keep it as TextField but then also just trim it if it's huge (~500 characters?) before saving it

Reply:
@nolachen Since the TextField is variable length, I wouldn't worry about it. My concern was allocating huge space for the column when it isn't needed but it seems Postgres doesn't do that.